sonar-html-plugin/src/main/java/org/sonar/plugins/html/HtmlPlugin.java

```diff
@@ -60,13 +64,21 @@ public void define(Context context) {
     return ImmutableList.of(
 
       PropertyDefinition.builder(HtmlConstants.FILE_EXTENSIONS_PROP_KEY)
-        .name("File suffixes")
+        .name("HTML File suffixes")
         .description("List of file suffixes that will be scanned.")
         .category(CATEGORY)
         .defaultValue(HtmlConstants.FILE_EXTENSIONS_DEF_VALUE)
         .deprecatedKey("sonar.web.file.suffixes")
         .onQualifiers(Qualifiers.PROJECT)
         .multiValues(true)
+        .build(),
+      PropertyDefinition.builder(HtmlConstants.JSP_FILE_EXTENSIONS_PROP_KEY)
+        .name("JSP File suffixes")
+        .description("List of file suffixes that will be scanned.")
```

Comment:
This description is identical to the the "HTML File suffixes" property: I think it would be better to make them different.